package org.cloudfoundry.credhub.exceptions;

public class MalformedPrivateKeyException extends RuntimeException {
}
